fabric/dataclient.go

```diff
@@ -446,13 +446,32 @@ func convertOne(fg *Fabric) ([]*eskip.Route, error) {
 				allowedServices := decideAllowedServices(fg.Spec.AllowList, m.AllowList)
 				if len(allowedServices) > 0 || allowedServices == nil {
 					r := createServiceRoute(m, eskipBackend, allowedOrigins, allowedServicesToFilterArgs(allowedServices), privs, fg.Metadata.Name, fg.Metadata.Namespace, host, p.Path)
+					if m.Response != nil {
+						patchRouteWithStaticResponse(r, m.Response)
+					}
+
 					routes = append(routes, r)
 
 					// ratelimit overrrides require separated routes with predicates.JWTPayloadAllKVName
 					if m.Ratelimit != nil && len(m.Ratelimit.Target) > 0 {
 						routes = append(routes, createRatelimitRoutes(r, m, fg.Metadata.Name, p.Path)...)
 					}
 				}
+
+				// routes to support x-fabric-employee-access
+				if m.EmployeeAccess != nil {
+					usersAllowed := make([]interface{}, 0, 2*len(m.EmployeeAccess.UserList))
+					sort.Strings(m.EmployeeAccess.UserList)
+					for _, u := range m.EmployeeAccess.UserList {
+						// TODO(sszuecs) should be configurable
+						usersAllowed = append(usersAllowed, "https://identity.zalando.com/managed-id", u)
+					}
+					rea := createEmployeeAccessRoute(m, eskipBackend, allowedOrigins, usersAllowed, m.EmployeeAccess.Type, fg.Metadata.Name, fg.Metadata.Namespace, host, p.Path)
+					if m.Response != nil {
```

Comment:
In connection with the previous comment - then we would not need to check the same if condition here.